Sources/BraveWallet/Crypto/Accounts/Add/AddAccountView.swift

```diff
@@ -80,7 +84,7 @@ struct AddAccountView: View {
     .listStyle(InsetGroupedListStyle())
     .listBackgroundColor(Color(UIColor.braveGroupedBackground))
     .navigationBarTitleDisplayMode(.inline)
-    .navigationTitle(Strings.Wallet.addAccountTitle)
+    .navigationTitle(String.localizedStringWithFormat(Strings.Wallet.addAccountTitle, preSelectedCoin?.localizedTitle ?? (selectedCoin?.localizedTitle ?? BraveWallet.CoinType.eth.localizedTitle)))
```

Comment:
This title doesn't work in the add account view coin selection screen.
![Simulator Screen Shot - iPhone 13 Pro - 2022-10-18 at 22 58 07](https://user-images.githubusercontent.com/5314553/196587375-d4869947-e5b5-422d-ad33-8a5c40c91cce.png)